src/index.html

```diff
@@ -5,9 +5,340 @@
+          </div>
+
+          <div class="testimonials__people ">
+            <img
+              src="/images/people/2.jpg"
+              alt="Photo of Scarlett Hawkins"
+              class="testimonials__photo"
+            >
+            <p class="testimonials__name">Scarlett Hawkins</p>
+            <p class="testimonials__function">Senior Vice President</p>
+            <p clss="testimonials__sign">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+          </div>
+
+          <div class="testimonials__people">
+            <img
+              src="/images/people/3.jpg"
+              alt="Photo of Max Henry"
+              class="testimonials__photo"
+            >
+            <p class="testimonials__name">Max Henry</p>
+            <p class="testimonials__function">Chief Design Officer</p>
+            <p clss="testimonials__sign">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="main__contacts contacts" id="contacts">
+        <div class="contacts__title title">
+          <p class="title__label">Contact us</p>
+          <img
+            src="/images/crown.svg"
+            alt="crown"
+            class="title__crown"
+          >
+        </div>
+        <div class="contacts__wrap">
+          <div class="contacts__info">
+            <div class="contacts__phone">
+              <p class="contacts__ph-label">Phone</p>
+              <a href="tel: +1 234 5555-55-55" class="contacts__ph-link">+1 234 5555-55-55</a>
+            </div>
+
+            <div class="contacts__email">
+              <p class="contacts__eml-label">Email</p>
+              <a href="#" class="contacts__eml-link">hello@miami.com</a>
+            </div>
+
+            <div class="contacts__address">
+              <p class="contacts__addr-label">Address</p>
+              <a href="#" class="contacts__addr-link">400 first ave. n. <br> suite 700 Minneapolis, MN 55401</a>
+            </div>
+          </div>
+
+          <div class="contact__data data-form">
+            <form action="POST" class="consultation__form data-form">
+              <input
+                type="text"
+                id="name_form"
+                placeholder="Name"
+                minlength="2"
+                maxlength="15"
+                required
+                class = "data-form__item"
+              >
+              <input
+                type="email"
+                id="email_form"
+                placeholder="Email"
+                required
+                class = "data-form__item"
+              >
+              <textarea
+                name="message"
+                id="message"
+                class="data-form__item data-form_big-sell"
+                placeholder="Message"
+              >
+              </textarea>
+              <input
```

Comment:
Якщо зробити так, то placeholder буде нормально відображатися при завантаженні сторінки а не як зараз його немає бо поле заповнене пробілами рядка. [Тут розбиралися чому так](https://mateacademy.slack.com/archives/G018RCJTGH2/p1599764543056900)))
```suggestion
              <textarea
                name="message"
                id="message"
                class="data-form__item data-form_big-sell"
                placeholder="Message"
             ></textarea>
              <input
```